src/parquet/column/test-util.h

```diff
@@ -316,7 +315,7 @@ template <typename Type>
 static shared_ptr<DictionaryPage> MakeDictPage(const ColumnDescriptor *d,
     const vector<typename Type::c_type>& values, const vector<int>& values_per_page,
     Encoding::type encoding, vector<shared_ptr<Buffer> >& rle_indices) {
-  InMemoryOutputStream page_stream;
+  InMemoryOutputStream page_stream(get_default_pool());
```

Comment:
I feel strongly against default args for memory pools - it's too easy to forget to specify a memory pool. However, these `get_default_pool()` do pollute the code - I'll try to come up with a better solution.